modules/dev-tools/src/helpers/get-config.js

```diff
@@ -18,7 +18,7 @@ configPath
   .split('.')
   .filter(Boolean)
   .forEach((path) => {
-    config = config[path];
+    config = config ? config[path] : undefined;
```

Comment:
I am not quite sure I understand what this does. Maybe add a comment, why we split on `.`?